### PR TITLE
[Fe] Issues Sign up errors

### LINKED ISSUE #115: [FE] - Display correct error messaging for signup errors

### PR DESCRIPTION
**Pull request by greenc123**
displays the correct error message if the email is not valid or there is no email or username

closes #115

### CODE REVIEW

**Comment by greenc123** [comment]
#120 
Screen Shot 2020-07-09 at 11 54 19 AM

Screen Shot 2020-07-09 at 11 54 33 AM


Screen Shot 2020-07-09 at 11 54 46 AM

File: src/routes/userRoutes.js [hunk 1]
```diff
@@ -54,9 +54,8 @@ routes.createUser = async (req, res) => {
     return res.status(200).json({ ...account.dataValues });
   } catch (err) {
     logger.error("Creating user failed", userInfo.username, err);
-    return res
-      .status(500)
-      .json({ error: { message: "Creating user failed. Please try again" } });
+    console.log("err", err.message);
```
Comment (songz) [inline src/routes/userRoutes.js:57]: use logger

File: src/server.js [hunk 2]
```diff
@@ -29,7 +28,7 @@ const startServer = async (portNumber) => {
     app.use(express.json());
     app.use(
       session({
-        secret: process.env.SECRET,
+        secret: "I L0V3 DATABASES",
```
Comment (songz) [inline src/server.js:31]: undo this page please. we want to use env variable.

File: views/signup.ejs [hunk 3]
```diff
@@ -46,8 +45,15 @@
         username: username.value.toLowerCase(),
         email: email.value.toLowerCase()
       })
-    }).then(r => r.json()).then(() => {
-      return location.href = '/signin'
+    }).then(r => r.json()).then((res) => {
+      console.log(res, 'response')
```
Comment (songz) [inline views/signup.ejs:49]: remove console.log

**Pull request merged**